client/rest-high-level/src/test/java/org/elasticsearch/client/SearchIT.java

```diff
@@ -269,7 +269,7 @@ public void testSearchWithMatrixStats() throws IOException {
         assertEquals(5, matrixStats.getFieldCount("num2"));
         assertEquals(29d, matrixStats.getMean("num2"), 0d);
         assertEquals(330d, matrixStats.getVariance("num2"), 0d);
-        assertEquals(-0.13568039346585542, matrixStats.getSkewness("num2"), 0d);
+        assertEquals(-0.13568039346585542, matrixStats.getSkewness("num2"), 1.0e-16);
```

Comment:
The matrixStats PR is here - https://github.com/elastic/elasticsearch/pull/18300 - with code from @nknize  et al.